index.js

```diff
@@ -10,8 +10,16 @@ module.exports = function (filepath, options) {
   var fullpath = (root) ? path.join(root, filepath) : filepath
 
   try {
-    return fs.statSync(fullpath).isFile()
-  } catch (e) {
-    return false
+    return fs.statSync(fullpath).isFile();
+  }
+  catch (e) {
+  
+    if (e.code == 'ENOENT') { // no such file or directory 
+      console.log("File does not exist.");
```

Comment:
Wops, this shouldn't be here. I just accidentally added while testing and forgot to delete it. Just `return false` statement in if and that's it. Thank you.